packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -733,6 +713,31 @@ export async function activate(extensionContext: vscode.ExtensionContext) {
   return api;
 }
 
+async function processPostProjectOpened(extensionContext: vscode.ExtensionContext) {
+  await workspaceContext.initialize(extensionContext);
+
+  // Register org picker commands
+  const orgList = new OrgList();
+  extensionContext.subscriptions.push(registerOrgPickerCommands(orgList));
+
+  await setupOrgBrowser(extensionContext);
+  await setupConflictView(extensionContext);
+
+  PersistentStorageService.initialize(extensionContext);
+
+  // Register file watcher for push or deploy on save
+  await registerPushOrDeployOnSave();
+  decorators.showOrg();
+  decorators.monitorOrgConfigChanges();
+
+  await setUpOrgExpirationWatcher();
```

Comment:
@randi274 Also, I initially had the impl for `setUpOrgExpirationWatcher()` in index.ts, but it wasn't testable (there are no _unit_ tests for the code in this index.ts... it's all tested _implicitly_ by the integration tests) so I moved setUpOrgExpirationWatcher() into its own file (packages/salesforcedx-vscode-core/src/util/orgUtil.ts) and created unit tests.